tests/config/android.config.ts

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+import { config as loadEnv } from 'dotenv';
+loadEnv({ path: path.join(__dirname, '..', '..', '.env') });
```

Comment:
Does this work if I don't have the `.env` file?

Reply:
Yes, otherwise the bots wouldn't have passed!